GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -573,11 +576,15 @@ private void InitializeComponent()
             // EditSettings
             // 
             this.EditSettings.DisplayStyle = System.Windows.Forms.ToolStripItemDisplayStyle.Image;
+            this.EditSettings.DropDownItems.AddRange(new System.Windows.Forms.ToolStripItem[] {
+                this.gitextSettingsToolStripMenuItem,
+                this.gitSettingsToolStripMenuItem,
+                this.pluginsSettingsToolStripMenuItem});
             this.EditSettings.Image = global::GitUI.Properties.Images.Settings;
             this.EditSettings.Name = "EditSettings";
-            this.EditSettings.Size = new System.Drawing.Size(23, 22);
+            this.EditSettings.Size = new System.Drawing.Size(32, 22);
             this.EditSettings.ToolTipText = "Settings";
-            this.EditSettings.Click += new System.EventHandler(this.OnShowSettingsClick);
+            this.EditSettings.ButtonClick += new System.EventHandler(this.OnShowSettingsClick);
```

Comment:
I guess this is the cause why the `Ctrl+,` hotkey does not work any longer.
`s/EditSettings.PerformClick/OpenSettings/`